### DISCUSSION TITLE
YITH WooCommerce Dynamic Pricing and Discounts

### DISCUSSION THREAD

**Discussion by osman-kocak** [inspireui/support#5548] [Working on Features]
YITH WooCommerce Dynamic Pricing and Discounts
![dynamic-screen](https://user-images.githubusercontent.com/57155882/172423483-067df693-1d13-4cb3-8227-c176be565fc6.png)

**Comment by Yan-Aleman** [comment]
❤️🔥🔥

**Comment by bishoynagyfakhry** [comment]
yes this is great need

**Comment by shahriar5653** [comment]
Anyone can integrate this plugin with my fluxstore woocommerce app? i will payment high rate

**Comment by medtr-git** [comment]
+1 
A very handy integration

**Comment by tryabp** [comment]
Much needed feature but i think [WooCommerce Dynamic Pricing & Discounts on codecanyon](https://codecanyon.net/item/woocommerce-dynamic-pricing-discounts/7119279) this plugin will work better than YITH.

**Comment by tryabp** [comment]
+1

**Comment by osman-kocak** [comment]
alternative plugin

[https://codecanyon.net/item/woocommerce-dynamic-pricing-discounts/7119279?gclid=CjwKCAiAp7GcBhA0EiwA9U0mtn5jXzmIPit5Q6SAuyMyWZrn6-agxKUFrgp7y33QcXlwIZOE2AkONhoC5ZYQAvD_BwE](https://codecanyon.net/item/woocommerce-dynamic-pricing-discounts/7119279?gclid=CjwKCAiAp7GcBhA0EiwA9U0mtn5jXzmIPit5Q6SAuyMyWZrn6-agxKUFrgp7y33QcXlwIZOE2AkONhoC5ZYQAvD_BwE)

**Comment by Yan-Aleman** [comment]
🔥🔥👌

**Comment by MarkHeaven** [comment]
When will this feature be available? Thank you)

**Reply by osman-kocak** [reply]
I hope it will be added soon

**Reply by DivitpatelMumbai** [reply]
A very important feature and any other similar plugin. With the ability to set dynamic prices for products.

Example:
https://woocommerce.com/products/dynamic-pricing/

**Comment by ElsaidMahmous** [comment]
need

**Comment by hassanyousufx** [comment]
When can this be done??

**Comment by osman-kocak** [comment]
https://codecanyon.net/item/woocommerce-dynamic-pricing-discounts/7119279

Check it please
@minhcasi

**Comment by devomman** [comment]
This must be good https://codecanyon.net/item/woocommerce-dynamic-pricing-discounts/7119279
